core/schema/src/main/liquibase/27.1.0/eventid-bigint.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd" >
+
+  <changeSet author="rangerrick" id="27.1.0-eventid-drop-related-views" runAlways="true">
+    <preConditions onFail="MARK_RAN">
+      <viewExists viewName="node_outages" />
+    </preConditions>
+
+    <!-- these will get recreated by views/*.xml -->
+    <dropView viewName="node_alarms" />
+    <dropView viewName="node_outages" />
+    <dropView viewName="node_outage_status" />
+  </changeSet>
```

Comment:
Minor thought: Do these views always go together? Would it be more robust to split it into 3 change sets?

Reply:
These views all reference `eventid` and thus have to be deleted before I can change `eventid`'s type (or else PostgreSQL complains about reference errors).  They get "normally" dropped/recreated in the other changes in `views/`